docs/website/src/site/styles/homepage/call-to-action.scss

```diff
@@ -296,6 +323,34 @@ $transition-timing: 200ms;
   }
 }
 
+@keyframes smileAnimationSafari {
+  30% {
+    transform: translateY(0);
+  }
+
+  60% {
+    transform: translateY(-5px);
+  }
+
+  100% {
+    transform: translateY(0);
+  }
+}
+
+@keyframes smileBottomAnimationSafari {
+  30% {
+    transform: translateY(0);
+  }
+
+  60% {
+    transform: translateY(-6px);
```

Comment:
can't you make the number of pixels a css variable and set that according to whether it's safari? It's a shame to have so many different animations when the only difference is 1px ;)